### ISSUE TITLE
Links in Summary do not have the right colors

### ISSUE THREAD

**Issue by onnyyonn** [onnyyonn/feed#4]
Even though we have the following style:
``` javascript
    .article summary a {
      color: var(--palette--frost);
    }
```
The links in Summary inherit the global style for links.

**Comment by 5310** [comment]
Ah, that would explain why the line-breaks are broken as well. That's the wrong class :awkwardwatchog:

**Comment by 5310** [comment]
#5 Actually closes this instead of #2 😬
